app/models/katello/glue/candlepin/consumer.rb

```diff
@@ -220,8 +203,6 @@ def save_candlepin_orchestration
         case orchestration_for
         when :hypervisor
           # it's already saved = do nothing
-        when :update
-          pre_queue.create(:name => "update candlepin consumer: #{self.name}", :priority => 3, :action => [self, :update_candlepin_consumer])
         end
       end
```

Comment:
Can't we delete this whole method 'save_candlepin_orchestration' and get rid of the before_save that calls it?